src/blocks/templates/blocks/logos.html

```diff
@@ -1,8 +1,5 @@
 {% load wagtailimages_tags %}
-<div class="container row">
-    {% if value.logos|length < 3 %}
-        <div class="col s12 m2"></div>
-    {% endif %}
+<div class="container row logo-container">
     {% for logo in value.logos %}
         <div class="col s12 m4 logo">
```

Comment:
Behöver vi fortfarande col s12 m4 om vi använder flexbox eller kan vi ta bort den?

Reply:
No, you're right, fixed now